x/concentrated-liquidity/swapstrategy/one_for_zero_test.go

```diff
@@ -152,7 +157,7 @@ func (suite *StrategyTestSuite) TestComputeSwapStepOutGivenIn_OneForZero() {
 	for name, tc := range tests {
 		tc := tc
 		suite.Run(name, func() {
-			strategy := suite.setupNewOneForZeroSwapStrategy(types.MaxSqrtPrice, tc.spreadFactor)
+			strategy := suite.setupNewOneForZeroSwapStrategy(types.MaxSqrtPrice, tc.spreadFactor, tc.tickSpacing)
```

Comment:
Can we just set the `daulttTickSpacing` here? It does not seem to matter for `ComputeSwapStepOutGivenIn`

Reply:
I think it can be removed from `oneForZeroStrategy` struct completely

Reply:
Good point on being able to remove it. 8f1647da0ac301ae79b9587e4806ab0aa561a0ea